blockly/src/utils/Direction.java

```diff
@@ -11,7 +11,10 @@ public enum Direction {
   /** The direction left. */
   LEFT("links", -1, 0),
   /** The direction right. */
-  RIGHT("rechts", 1, 0);
+  RIGHT("rechts", 1, 0),
+
+  /** The current Position. */
+  HERE("hier", 0, 0);
```

Comment:
Eine Richtung, die keine ist? Können wir nicht einfach `null` verwenden? Quasi wenn keine Richtung (`null`), dann gucke auf den aktuellen Tile.

Reply:
Puhh pähhh find ich jetzt auch nicht knorke. 
Hmpf

Reply:
ja, aber ich denke auch etwas an #1798 . Wenn wir dann am Ende eine `Direction` im Core haben, dann sollte es keine "keine Richtung"

Reply:
Vielleicht müssen wir Direction rename? move-Vektor? Idk

Reply:
Der Vorteil an der 0,0 Direction ist halt, das du keine Fallunterscheidung machen musst, z.B in `move`. 
```
        comp.vc.currentXVelocity(direction.x() * comp.vc.xVelocity());
        comp.vc.currentYVelocity(direction.y() * comp.vc.yVelocity());
        
```

Ein `null` würde dir hier platzen, musst du also vorher prüfen.


Ich glaube wenn `Direction` in `DirectionVector` umbenannt werden würde, wäre ein `0,0` Vektor auch nicht mehr verwirrend/unerwartet/seltsam/außenseiter.

Reply:
Dann muss die Person gucken die dann das clean up issue bearbeitet